index.html

```diff
@@ -27,6 +26,141 @@ <h1 class="section-header" id="my-name">Christian Onoh</h1>
       <p class="my-button">Start collaboration</p>
       <img class="project-img" src="assets/images/image_geometry_2.svg" alt="">
```

Comment:
- [ ] I think it would be better to add the proper background image by using ```CSS background-image``` Kindly it is recommended that decorations are backgrounded images, not ```img``` tags 🙏